src/plugins/intel_gpu/include/intel_gpu/graph/network.hpp

```diff
@@ -125,10 +125,27 @@ struct network {
             evt = get_primitive_event(output_id);
         return network_output(evt, get_output_memory(output_id), get_stream_ptr());
     }
-
-    memory::ptr get_output_memory(const primitive_id& output_id);
     layout get_node_output_layout(const primitive_id& output_id) const;
-    layout get_output_layout(const primitive_id& output_id) const;
+
+    template <class T, class U = T>
+    std::vector<U> get_output_values(const primitive_id& id, size_t max_cnt = std::numeric_limits<size_t>::max()) {
+        std::vector<U> ret;
+        auto ptr = get_output_memory(id);
+        auto out_ids = get_output_ids();
+        if (find(out_ids.begin(), out_ids.end(), id) == out_ids.end() && get_engine().configuration().use_memory_pool)
```

Comment:
Let's remove this condition `get_engine().configuration().use_memory_pool`

Reply:
Done.